src/app/client/src/app/modules/observation/components/observation-listing/observation-listing.component.html

```diff
@@ -1,6 +1,6 @@
 <app-landing-section [textToDisplay]="resourceService?.frmelmnts?.lbl?.observation" [svgToDisplay]="svgToDisplay"
```

Comment:
not required. you can remove

Reply:
anyway we are not working on the observation module as per the suggestion provided by vinu kumar